Tests/ForageSDKTests/ForagePANTextFieldTests.swift

```diff
@@ -12,16 +12,13 @@ final class ForagePANTextFieldTests: XCTestCase {
     var foragePANTextField: ForagePANTextField!
     
     override func setUp() {
-         ForageSDK.setup(ForageSDK.Config(
-            environment: .sandbox,
+        ForageSDK.setup(ForageSDK.Config(
             merchantID: "merchantID123",
             sessionToken: "authToken123"
         ))
         ForageSDK.shared.service = nil
         ForageSDK.shared.panNumber = ""
-        foragePANTextField = ForagePANTextField()
-       
-        observableState = nil
```

Comment:
Why was setting the `observableState = nil` line removed?

Reply:
It was breaking the unit tests, it seems like `observableState` is no longer a member in this test suite